.github/workflows/abdullahCode.py

```diff
@@ -0,0 +1,225 @@
+print("Hello this is a program that does math for you, press enter after every answer") 
+
+print("")
```

Comment:
0072059@schoolsnet.act.edu.au
why this print statement


why this print statement